src/contrib/cluster/Akka.Cluster.Metrics/Serialization/EWMA.cs

```diff
@@ -31,8 +31,12 @@ public static partial class Types
             ///             the sampled value resulting from the previous smoothing iteration.
             ///             This value is always used as the previous EWMA to calculate the new EWMA.
             /// </summary>
-            public sealed partial class EWMA 
+            public sealed class EWMA: IEquatable<EWMA>
             {
+                public double Value { get; }
+                
+                public double Alpha { get; }
+                
```

Comment:
Added equality code and missing fields

Reply:
Note that in the old code, IEquatable was implemented by the protobuf generated code. We need to add this back.